src/Perlin.tsx

```diff
@@ -10,8 +10,10 @@ import { SmartBuffer } from "smart-buffer";
 import PayloadReader from "./payload/PayloadReader";
 import { IAccount } from "./types/Account";
 import ReconnectingWebSocket from "reconnecting-websocket";
+import { Wavelet, Contract, TAG_TRANSFER } from "wavelet-client";
+import JSBI from "jsbi";
 // @ts-ignore
-import * as JSONbig from "json-bigint";
+window.useNativeBigIntsIfAvailable = false;
```

Comment:
Just curious, any reason why we shouldn't opt in to using native big ints if they are available? Would it break any sort of compatibility with JSBI?

Reply:
Setting it to true actually cased an error from wavelet-client. 
The error is fixed now under https://github.com/perlin-network/wavelet-client-js/pull/6 and can set this to true afterwards.